### ISSUE TITLE
Flux can't change prompt during training

### ISSUE THREAD

**Issue by jpXerxes** [kohya-ss/sd-scripts#1513]
bmaltais says this is a problem in sd-scripts

Changing the prompt.txt during training produces the following message when the next sample point is reached, at which point it hangs. I like to change the prompt to see various aspects as the training proceeds, but can't with Flux loras.

(path)\venv\lib\site-packages\torch\utils\checkpoint.py:61: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
warnings.warn(
